client/my-sites/domains/domain-management/dns/mx-record.jsx

```diff
@@ -14,56 +15,71 @@ import FormTextInput from 'components/forms/form-text-input';
 import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
 import FormSettingExplanation from 'components/forms/form-setting-explanation' ;
 
-const MxRecord = React.createClass( {
-	statics: {
-		initialFields: {
-			name: '',
-			data: '',
-			aux: 10
-		}
-	},
+class MxRecord extends React.PureComponent {
+	static initialFields = {
+		name: '',
+		data: '',
+		aux: 10
+	};
 
-	propTypes: {
+	static propTypes = {
 		fieldValues: React.PropTypes.object.isRequired,
 		onChange: React.PropTypes.func.isRequired,
 		selectedDomainName: React.PropTypes.string.isRequired,
 		show: React.PropTypes.bool.isRequired
-	},
-
-	fieldNames: [ 'name', 'data', 'aux' ],
+	};
 
 	render() {
```

Comment:
Seems the root cause is that we don't initialize (in `state`) all the fields we then use to control the inputs. For example, `aux` is missing.